AfricanReligions/events/oldgods_ancestor_worship_events.txt

```diff
@@ -2509,9 +2567,88 @@ character_event = {
 				}
 				#Hunting dieties
 				if = {
-					limit = { has_character_modifier = follower_of_taipo_and_mielikki }
+					limit = { 
+						OR = {
+							has_character_modifier = follower_of_taipo_and_mielikki 
+							has_character_modifier = follower_of_tan_dowa
+						}
+					}
 					character_event = { id = TOG.7517 }
 				} 
+				#Call Yan Riga flavour event
+				if = {
+					limit = { has_character_modifier = follower_of_yan_riga }
+					character_event = { id = AFRREL.106 }
+				}
+				#Call Babbaku flavour event
+				if = {
+					limit = { has_character_modifier = follower_of_babbaku }
+					character_event = { id = AFRREL.107 }
+				}
+				#Call Tan Garki flavour event
+				if = {
+					limit = { has_character_modifier = follower_of_tan_garki }
+					character_event = { id = AFRREL.110 }
+				}
+				#Call Yayan Zanzanna flavour event
+				if = {
+					limit = { 
+						AND = {
+							has_character_modifier = follower_of_yayan_zanzanna
+							OR = {
+								has_disease_trigger = yes
+								trait = incapable
+							}
+						}
+					}
+					character_event = { id = AFRREL.111 }
+				}
+				#Call Tan Dowa flavour event
+				if = {
+					limit = { has_character_modifier = follower_of_tan_dowa }
+					character_event = { id = AFRREL.112 }
+				}
+				#Call Anti flavour event
+				if = {
+					limit = { 
+						AND = {
+							has_character_modifier = follower_of_anti
+							is_married = yes
+						}
+					}
+					random_spouse = {
+						limit = {
+							OR = {
+								is_female = no
+								AND = {
+									is_female = yes
+									NOT = { age = 46 }
```

Comment:
The vanilla events I looked at had this restriction for most romantic/sexual events for women. I don't see the point I just kept it for consistency

Reply:
It's probably because there's a chance or a guarantee of pregnancy being scripted.  Since that's not the case for this one, I don't think there should be an age restriction.